src/main/java/org/spongepowered/api/text/Text.java

```diff
@@ -54,18 +64,34 @@
  * also in other places like books or signs.</p>
  *
  * <p>Text instances can be either directly created through the available
- * constructor or using the {@link TextBuilder} available through one of the
- * {@link Texts#builder()} methods, which is the recommended way.</p>
+ * constructor or using the {@link Builder} available through one of the
+ * {@link Text#builder()} methods, which is the recommended way.</p>
  *
- * @see Texts#builder()
- * @see TextBuilder
- * @see Literal
- * @see Translatable
- * @see Selector
- * @see Score
+ * @see Text#builder()
+ * @see Builder
+ * @see LiteralText
+ * @see TranslatableText
+ * @see SelectorText
+ * @see ScoreText
  */
 public abstract class Text implements TextRepresentable {
 
+    /**
+     * The empty, unformatted {@link Text} instance.
+     */
+    public static final Text EMPTY = LiteralText.EMPTY;
+
+    /**
+     * An unformatted {@link Text} that will start a new line (if supported).
+     */
+    public static final LiteralText NEW_LINE = new LiteralText("\n");
```

Comment:
Yeah, but the LiteralText.Builder will automatically point to this one. 
=>
`CommandMessageFormatting.NEWLINE_TEXT` <= `Text.NEW_LINE`

[LiteralText.Builder#build()](https://github.com/SpongePowered/SpongeAPI/pull/988/files#diff-f4d8dd0ce6550f824748066654468e58R198)